### DISCUSSION TITLE
A spanish Lenguaje and themes?

### DISCUSSION THREAD

**Discussion by katgold87** [OnionUI/Onion#521] [General]
Hello! I update a onion os 3.10.5 and the lenguaje spanish not disponible and the app custom themes either.
How do i solve?
Thanks a lot.

**Comment by Sichroteph** [comment]
Would you try to make a Spanish translation file ?

**Reply by vics3g** [reply]
[lang.zip](https://github.com/jimgraygit/Onion/files/8845268/lang.zip)
ERASE es.lang and es.lang.short from :/miyoo/app/lang to solve

And please developers solve....

**Reply by Sichroteph** [reply]
Thanks 👍
